docs/tools/new_tools_temp.md

```diff
@@ -0,0 +1,53 @@
+# New tools - draft
+
+<!--well this is a disaster-->
+
+Starting with version 6.x, Mbed OS is moving from Mbed CLI to a new build tool: it's cmake-based,
+
+You'll still need Mbed CLI for older versions of Mbed OS. You can install both tools side by side.<!--seems like Mbed CLI also requires Python 3, so there shouldn't be any issues-->
+
+
+# Install or upgrade
+
+Mbed Tools is Python based, so you can install it with pip.
```

Comment:
"The Mbed Tools are Python based"?

Reply:
But it's a single tool. 
Which is one of the problems I had with the name: It's a plural name but there's just one tool. 
Hence waiting on Andy.

Reply:
"The new command line tool is a Python package named `mbed-tools`, so you can install it with pip."?

Reply:
We're gonna wait for a proper decision about name